pyproject.toml

```diff
@@ -75,6 +75,7 @@ vulture = "^2.6"
 [tool.poetry.dev-dependencies]
 scan-build = "*"
 towncrier = "^21.9.0"
+#freetype-py = "^2.3.0"
```

Comment:
Does it belong here?

Reply:
I always forget the name of the package when i need to regenerate fonts. Will add it as a real dependency after #2890 is merged.